behave/features/steps/common_steps.py

```diff
@@ -302,6 +302,7 @@ def check_accessibility(context):
     axe.inject()
     results = axe.run()
     if len(results["violations"]) > 0:
-        make_dir("feature_errors")
-        axe.write_results(results, "feature_errors/ally.json")
+        a11y_error_dir = os.path.join(context.artifacts_dir, "feature_errors")
+        make_dir(a11y_error_dir)
```

Comment:
do we need the a11y_error_dir variable? 
does `make_dir(os.path.join(context.artifacts_dir, "feature_errors"))` work?

Reply:
That wont work, it only takes one parameter. 

We need `a11y_error_dir` as the file structure changes when it's in different environments. Without this it will check the wrong file path.